drivers/espi/CMakeLists.txt

```diff
@@ -2,3 +2,5 @@
 
```

Comment:
Commit title: "drivers: espi: Add driver for microchip XEC family"

Reply:
These are the generic definitions for all eSPI drivers. I think you meant the commit after this